style.css

```diff
@@ -115,455 +115,3 @@ a {
   color: white;
   margin-top: 1.2rem;
 }
```

Comment:
- [ ] Kindly add style to remove the dots from the list, to be like the design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/154340746-02f4a567-ee14-4722-bf4b-4b6d7e9ac818.png) | ![image](https://user-images.githubusercontent.com/35535130/154340850-eeb3150b-f2d9-4bb9-b0dc-0bc660a6b2e6.png) |